plugins/Kaleidoscope-Colormap-Overlay/src/kaleidoscope/plugin/Colormap-Overlay.cpp

```diff
@@ -30,13 +30,20 @@
 #include "kaleidoscope/key_defs.h"              // for Key, KEY_FLAGS, Key_NoKey, LockLayer
 #include "kaleidoscope/layers.h"                // for Layer, Layer_
 #include "kaleidoscope/plugin/LEDControl.h"     // for LEDControl
+#include <Kaleidoscope-FocusSerial.h>           // for Focus
 #include <Kaleidoscope-LED-Palette-Theme.h>     // for LEDPaletteTheme
 
 namespace kaleidoscope {
 namespace plugin {
 uint16_t ColormapOverlay::map_base_;
 
 void ColormapOverlay::setup() {
+  // It appears that a call to ::LEDPaletteTheme.reserveThemes() is needed
+  // because it's where palette_base_ gets initialized. Since this plugin (and
+  // possibly others) don't actually use themes, requesting memory for storing
+  // themes doesn't make much sense. Maybe initialisation of palette_base_
+  // could be moved to a setup() method, though maybe the palette should be
+  // split from palette theme altogether?
```

Comment:
@obra I don't really like having to reserve memory for a theme that I don't even use. I'm happy to remove the comment and just accept it if that's what you prefer, but as stated in the comment I feel it might be better to split initialisation of the palette and logic for themes.

Reply:
I don't know that it should be in a 'setup' method? But possibly in a 'reservePalette' method (that the existing 'reserveThemes' would be refactored to call)?

Reply:
That's fine by me. Want a separate PR for that too?

Reply:
#1482